anki/schedv2.py

```diff
@@ -1581,41 +1582,41 @@ def maybeRandomizeDeck(self, did=None):
     def _emptyAllFiltered(self):
         self.col.db.execute("""
 update cards set did = odid, queue = (case
-when type = 1 then 0
-when type = 3 then 2
+when type = %d then %d
```

Comment:
If you are going to switch to constants rather than 'magic numbers', then I think it would be preferable to use named placeholders in format strings like this (e.g. `%(CARD_LRN)d`). Otherwise I think that you lose clarity rather than gaining it.

Reply:
If @dae thinks that one day, he may accept this pull request, then I'll follow your advice. 
Since it seems he is not interested I won't do more works on this. It is more clear for me at least, right now, and I may well be the only one to read this.

Reply:
I tried to use a style consistent with current code. Otherwise, I would have used formatted litterals, do f"{CARD_LRN}"

Reply:
Please note for future pull requests that the anki/ part of the codebase currently needs to support Python 3.5, so format literals can not be used.